lib/src/component_declaration/component_base.dart

```diff
@@ -459,13 +459,7 @@ abstract class UiStatefulComponent<TProps extends UiProps, TState extends UiStat
 ///
 /// > Note: Implements [MapViewMixin] instead of extending it so that the abstract state declarations
 /// don't need a constructor. The generated implementations can mix that functionality in.
-abstract class UiState extends _OverReactMapView implements StateMapViewMixin {
-  // Manually implement members from `StateMapViewMixin`,
-  // since mixing that class in doesn't play well with the DDC.
-  // TODO find out root cause and reduced test case.
-  @override Map get _map => this.state;
-  @override String toString() => '$runtimeType: ${prettyPrintMap(_map)}';
```

Comment:
@greglittlefield-wf I'm assuming that `toString` still returns `'$runtimeType: ${prettyPrintMap(_map)}'` in the super class?

Reply:
Yup, that should get mixed in with `PropsMapViewMixin`